addons/mail/static/src/components/composer_suggested_recipient/composer_suggested_recipient.js

```diff
@@ -128,7 +130,9 @@ class ComposerSuggestedRecipient extends Component {
                 this._dialogRef.comp.widget.on('closed', this, () => {
                     this._isDialogOpen = false;
                 });
-                this._dialogRef.comp.widget.open();
+                var widget = this._dialogRef.comp.widget;
+                widget.context = Object.assign({}, widget.context, session.user_context)
```

Comment:
It's strange to have to do this here specifically, I wonder if it shouldn't be part of `FormViewDialogComponentAdapter` or genrally `ComponentAdapter`, but I guess it's fine here as a small fix.

Reply:
Here we add to the widget some user context. User context may or may not be used by other components. If not used it is just a few small variables.